packages/action-group/stories/action-group-tooltip.stories.ts

```diff
@@ -141,25 +156,36 @@ const template = (args: Properties): TemplateResult => {
             }}
         >
             <overlay-trigger>
-                <sp-action-button slot="trigger">Red</sp-action-button>
+                <sp-action-button slot="trigger" size=${args.size}>
```

Comment:
I think you mean for this to be on the `<sp-action-group>` on like 148 instead of on these individually, right?